database/seeders/ConstantGameSeeder.php

```diff
@@ -29,6 +29,7 @@ public function run()
         ]);
 
         $game = Game::create([
+            'name' => 'Game 1',
```

Comment:
Je zou in de `seeder` ook de `factory` van `Game` kunnen toepassen

Reply:
Dit kan niet vanwege problemen tussen Heroku en Factories